.update-changes.cfg

```diff
@@ -1,4 +1,4 @@
-new_commit_msg="Update CHANGES. [skip ci]"
+new_commit_msg="Update CHANGES."
```

Comment:
Unintended change?

Reply:
Yes, this slipped in when I cut the release.